### PR TITLE
unit test for PR

### PR DESCRIPTION
**Pull request by siarotchi**

### CODE REVIEW

File: src/components/PageTitle/PageTitle.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  it("should match the snapshot", () => {
+    /**
+     * Write a snapsho◘t test for PageTitle component
+     * Don't forget to add snapshot file to the git
+     */
+
+    const labelText = "label text";
+    const { container, getByText } = render(<PageTitle label={labelText} />);
+
+    expect(getByText(labelText)).toBeInTheDocument();
```
Comment (msuhov) [inline src/components/PageTitle/PageTitle.test.js:15]: You have already checked that label was rendered via snapshot test, so it is not necessary to check the labelText - it's just a double work, or you do a snapshot or you check by queries

File: src/components/PageTitle/__snapshots__/PageTitle.test.js.snap [hunk 2]
```diff
@@ -0,0 +1,12 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<PageTitle /> should match the snapshot 1`] = `
+<div>
```
Comment (msuhov) [inline src/components/PageTitle/__snapshots__/PageTitle.test.js.snap:4]: Not a big deal, but in the snapshot you see this `div`, which is not present in `PageTitle` component
This div is used internally by the library, and to make the snapshot shorter and cleaner we can do it not from the whole `container` object, but from it's `firstChild`

```
const { container: { firstChild } } = render(..)
expect(firstChild).toMatchSnapshot()
```

File: src/reducers/user-reducer.test.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        id: "123456",
+        name: "User name",
+        email: "smth@test.com",
+      },
+      abc12: {
+        id: "abc12",
+        name: "John Doe",
+        email: "world@test.com",
+      },
+      r2d2: {
```
Comment (msuhov) [inline src/reducers/user-reducer.test.js:42]: R2-D2 - nice 😄

File: src/components/PageTitle/PageTitle.test.js (hunk 1 above)
Comment (siarotchi) [reply src/components/PageTitle/PageTitle.test.js:15]: got it. Tx

**Pull request merged**